src/xdocs/config_filters.xml

```diff
@@ -410,6 +410,8 @@ public class UserService {
             document</em></a> in a file. If there is no configured
             suppressions file or the optional is set to true and
             suppressions file was not found the Filter accepts all audit events.
+            Filter processes only line-column suppressions from suppressions file, with tag
```

Comment:
comment out all documentation in xdocs as we cannot show it to users, because the whole model is not ready. We will uncomment the documentation in the future.

Reply:
done